next/components/ui/FeaturedBlogs/FeaturedBlogs.tsx

```diff
@@ -17,13 +17,15 @@ export const FeaturedBlogs = ({ blogs }: FeaturedBlogsProps) => {
         </div>
         <div className="flex w-1/2 flex-col gap-y-8">
           {blogs.slice(1, 3).map((blog, index) => (
-            <BlogSearchCard key={index} className="h-52" imageClassName="w-52" item={blog} />
+            <BlogSearchCard key={index} className="h-52" imageClassName="w-[206px]" item={blog.data} />
           ))}
         </div>
       </div>
-      <HorizontalScrollWrapper className="gap-x-4 px-8 lg:hidden">
+      <HorizontalScrollWrapper className="px-7.5 gap-x-4 lg:hidden">
```

Comment:
7.5 does not exist (anymore), would result in zero padding

```suggestion
      <HorizontalScrollWrapper className="px-8 gap-x-4 lg:hidden">
```